vpr/src/route/clock_network_builders.cpp

```diff
@@ -245,6 +245,8 @@ void ClockRib::create_rr_nodes_and_internal_edges_for_one_instance(ClockRRGraphB
     VTR_ASSERT(repeat.y > 0);
     VTR_ASSERT(repeat.x > 0);
 
+    int layer_num = 0; //Function *FOR NOW* assumes that layer is always 0
```

Comment:
Will probably want a To do here ... this is rr-graph code, so build on any layer with an rr-graph.